cim-client-sdk/src/main/java/com/crossoverjie/cim/client/sdk/impl/ClientBuilderImpl.java

```diff
@@ -5,12 +5,20 @@
 import com.crossoverjie.cim.client.sdk.Event;
 import com.crossoverjie.cim.client.sdk.io.MessageListener;
 import com.crossoverjie.cim.client.sdk.io.ReconnectCheck;
+import com.crossoverjie.cim.client.sdk.io.backoff.BackoffStrategy;
+import com.crossoverjie.cim.client.sdk.io.backoff.RandomBackoff;
 import com.crossoverjie.cim.common.util.StringUtil;
+
+import java.lang.reflect.Constructor;
+import java.util.ServiceLoader;
 import java.util.concurrent.ThreadPoolExecutor;
 import okhttp3.OkHttpClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class ClientBuilderImpl implements ClientBuilder {
 
+    private Logger logger = LoggerFactory.getLogger(ClientBuilderImpl.class);
```

Comment:
```suggestion
```

Why add this line of code?  if you need to record log,  it's recommended to use `@Slf4j` annotation.